src/components/Card.js

```diff
@@ -1,36 +1,95 @@
 import React, { Component } from 'react'
 import './Style.css'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Link } from "react-router-dom";
+import Pop from './Pop';
 
 class Card extends Component {
+
+    //Function to display right amount stars according to the rate
+    displayStars = (rate) => {
```

Comment:
this function will be run in any evaluation to the component so the most important to use useMemo for improving performance

Reply:
Done